src/flint/types/fmpz_mod.pyx

```diff
@@ -67,7 +67,7 @@ cdef class fmpz_mod_ctx:
         Return the modulus from the context as an fmpz
         type
 
-            >>> mod_ctx = fmpz_mod_ctx(2**127 - 1)
+            >>> mod_ctx = fmpz_mod_poly_ctx(2**127 - 1)
```

Comment:
I don't think this should be changed.

Reply:
You are right, I obviously got confused while refactoring. Thank you!